_posts/en/newsletters/2021-06-09-newsletter.md

```diff
@@ -0,0 +1,211 @@
+  these keys correctly, it was recently [discovered][btcd issue] that some other
+  wallets were incorrectly deriving slightly more than 1 out of 128 keys
+  incorrectly by failing to pad extended private keys (xprivs) that were
+  less than 32 bytes long.  This doesn't directly result in a loss of
+  funds or a reduction in security, but it does create problems for
+  users who create an HD wallet seed in one wallet and import it into
+  another wallet or who create multisignature wallets.  The test vector
+  implemented in this PR is also being [added][bips #1030] to BIP32 to
+  help future wallet authors avoid this issue.
+
+- [C-Lightning #4532][] adds experimental support for [upgrading a
+  channel][bolts #868]---rebuilding the latest commitment transaction so
+  that it can incorporate new features or structural changes, e.g.
+  converting to using [taproot][topic taproot].  The protocol starts
+  with a request for [quiescence][bolts #869], an agreement that neither
+  party will send any new state updates until the quiessence period is
+  completed.  During this period, the nodes negotiate the changes they
+  want to make and implement them.  Finally, the channel is restored to
+  full operation.  C-Lightning currently implements this during
+  connection reestablishment when the channel has already been in a
+  period of forced inactivity.  Various proposals for channel upgrades
+  were discussed in [Newsletter #108][news108 channel upgrades] and the
+  author of this PR wants the feature in part to work on the "simplified
+  HTLC negotiation" described in [Newsletter #109][news109 simplified
+  htlc].  This particular PR allows upgrading old channels to support
+  `option_static_remotekey`, which C-Lightning first added support for
+  in 2019, see [Newsletter #64][news64 static remotekey].
+
+- [LND #5336][] lnrpc: allow AMP pay-addr override + bump invoice timeouts FIXME:dongcarl
+
+- [BTCPay Server #2474][] adds the ability to test webhooks by sending
+  fake events that contain all their normal fields but dummy data.  This
+  mirrors testing features available on centrally hosted Bitcoin payment
+  processors such as Stripe and Coinbase Commerce.
+
+<!-- FIXME:harding to update releases and RCs, plus add topic links -->
+{% include references.md %}
+{% include linkers/issues.md issues="22051,22050,22095,4532,869,5336,2474,1030,868,21767" %}
+[LND 0.13.0-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.13.0-beta.rc3
+[news64 static remotekey]: /en/newsletters/2019/09/18/#c-lightning-3010
+[news108 channel upgrades]: /en/newsletters/2020/07/29/#upgrading-channel-commitment-formats
+[news109 simplified htlc]: /en/newsletters/2020/10/21/#simplified-htlc-negotiation
+[zmn ff]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-April/001986.html
+[zmn revive]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2021-May/003038.html
+[zmn preso]: https://zmnscpxj.github.io/offchain/2021-06-fast-forwards.odp
```

Comment:
This begins downloading locally on click. I assume there aren't any hosted versions we could link to instead?

Reply:
That's the link Zmn provided in his email.  If we really wanted, we could import the file to some webapp---Zmn was kind enough to put a liberal license on the file---but I'd prefer to avoid doing that unless necessary.